src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -566,10 +566,7 @@
             </form>
             <br>
             <br>
-          </div>
```

Comment:
It is line 519 that should be deleted instead. The problem was introduced [here](https://github.com/TEAMMATES/teammates/pull/8494/files#diff-2e74ce40cd9ed6f73d86dc9bc60dab4e) (but I failed to catch it then 😅).

Reply:
Done :smile: